src/main/java/net/consensys/orion/cmd/Orion.java

```diff
@@ -179,6 +180,8 @@ public static void configureRoutes(
         .handler(new ReceiveHandler(enclave, storage, APPLICATION_OCTET_STREAM));
     clientRouter.post("/privacyGroupId").consumes(JSON.httpHeaderValue).produces(JSON.httpHeaderValue).handler(
         new PrivacyGroupHandler(privacyGroupStorage));
+    clientRouter.post("/deletePrivacyGroupId").consumes(JSON.httpHeaderValue).produces(JSON.httpHeaderValue).handler(
```

Comment:
Have we considered implementing this as handling a `delete` verb instead of a `post` with the same path `/privacyGroupId`?
What do you guys think?

Reply:
agreed